tests/foreman/cli/test_operatingsystem.py

```diff
@@ -373,3 +383,36 @@ def test_positive_update_parameters_attributes(self):
         os = OperatingSys.info({'id': os_id}, output_format='json')
         self.assertEqual(param_name, os['parameters'][0]['name'])
         self.assertEqual(param_value, os['parameters'][0]['value'])
+
+    @tier4
```

Comment:
you can keep either tier4 or destructive. Not both.

Reply:
okay